src/server/req_modify.c

```diff
@@ -151,6 +151,7 @@ req_modifyjob(struct batch_request *preq)
 	int		 bad = 0;
 	int		 i;
 	int		 jt;		/* job type */
+	int      js;		/* job state */
```

Comment:
Align the variable declaration with the previous ones.

Reply:
Fixed in rc 17 https://github.com/PBSPro/pbspro/pull/590/commits/f94a3405ef13edaa87df40a89af36420b9627c7b